docker-compose-lf.yml

```diff
@@ -17,7 +17,7 @@ services:
         volumes:
         - .:/workspace/agent/
         image: duckietown/gym-duckietown-agent:gpu
-        command: /usr/local/bin/python agent/agent.py --no-render
+        command: /opt/conda/bin/python agent/agent.py --no-render
```

Comment:
@julian121266 why the change to a `conda`-oriented installation?

Reply:
Not sure there is not another solution, but without this change I get:

Recreating gym-duckietown-agent_gym-duckietown-agent_1  ... error

ERROR: for gym-duckietown-agent_gym-duckietown-agent_1  Cannot start service gym-duckietown-agent: OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \"/usr/local/bin/python\": stat /usr/local/bin/python: no such file or directory": unknown

ERROR: for gym-duckietown-agent  Cannot start service gym-duckietown-agent: OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \"/usr/local/bin/python\": stat /usr/local/bin/python: no such file or directory": unknown
ERROR: Encountered errors while bringing up the project.

Reply:
Why not just `python`? I think that's one of the advantages of the containerization.